mne_bids/write.py

```diff
@@ -57,6 +57,14 @@
                              ANONYMIZED_JSON_KEY_WHITELIST)
 
 
+class AnonymizeArg(TypedDict, total=False):
```

Comment:
Suggestion:
make it plural and private

```suggestion
class _AnonymizeArgs(TypedDict, total=False):
```